snuba/datasets/groups.py

```diff
@@ -44,6 +45,9 @@ def get_table_writer(self) -> Optional[TableWriter]:
     def get_query_processors(self) -> Sequence[QueryProcessor]:
         return [SimpleJoinOptimizer(), PrewhereProcessor()]
 
+    def get_query_splitters(self) -> Sequence[StorageQuerySplitStrategy]:
+        return []
+
```

Comment:
Splitting a join query is a more complex issue. We cannot simply take the splitters from the storages we are joining and apply all of them. In most cases the query would not even work.
I think the decision belongs to the JoinQueryPlanBuilder (that does not yet exists). It knows the entities we are joining, it knows which storages have been selected, it knows which type of joins we are applying, so it can decide if any split strategy can be applied.
Hopefully we will be able to make this process generic and not custom per dataset.